.azure/scripts/setup-kubernetes.sh

```diff
@@ -89,7 +89,14 @@ if [ "$TEST_CLUSTER" = "minikube" ]; then
       set -ex
     fi
 
-    if [ "$ARCH" = "s390x" ] || [ "$ARCH" = "ppc64le" ]; then
+    if [ "$ARCH" = "s390x" ]; then
+        git clone -b v1.9.11 --depth 1 https://github.com/kubernetes/kubernetes.git
+        sed -i 's/:1.11/:1.22.1/' kubernetes/cluster/addons/registry/images/Dockerfile
+        docker build --pull -t gcr.io/google_containers/kube-registry-proxy:0.4-${ARCH} kubernetes/cluster/addons/registry/images/
+        minikube image load ${ARCH}/registry:2.8.0-beta.1 gcr.io/google_containers/kube-registry-proxy:0.4-${ARCH}
+        minikube addons enable registry --images="Registry=${ARCH}/registry:2.8.0-beta.1,KubeRegistryProxy=google_containers/kube-registry-proxy:0.4-${ARCH}"
+        rm -rf kubernetes
+    elif [[ "$ARCH" = "ppc64le" ]]; then
         git clone -b v1.9.11 --depth 1 https://github.com/kubernetes/kubernetes.git
         sed -i 's/:1.11//' kubernetes/cluster/addons/registry/images/Dockerfile
```

Comment:
maybe you should also update version here.

Reply:
issue is same, this is reason why we also changed version for arm64 registry.

Reply:
We do not have testing environment ppc64le.